src/orb.yml.hbs

```diff
@@ -550,17 +605,27 @@ commands:
       - run:
           name: Update service with registered task definition
           command: |
-            SERVICE_NAME="$(echo << parameters.service-name >>)"
+            DEPLOYMENT_CONTROLLER="$(echo << parameters.deployment-controller >>)"
 
-            if [ -z "${SERVICE_NAME}" ]; then
-                SERVICE_NAME="$(echo << parameters.family >>)"
+            if [ "${DEPLOYMENT_CONTROLLER}" = "CODE_DEPLOY" ]; then
+                DEPLOYED_REVISION=$(aws deploy create-deployment \
+                    --application-name "<< parameters.application-name >>" \
+                    --deployment-group-name "<< parameters.deployment-group-name >>" \
+                    --revision '{"revisionType": "AppSpecContent", "appSpecContent": {"content": "{\"version\": 1, \"Resources\": [{\"TargetService\": {\"Type\": \"AWS::ECS::Service\", \"Properties\": {\"TaskDefinition\": \"'${CCI_ORB_AWS_ECS_REGISTERED_TASK_DFN}'\", \"LoadBalancerInfo\": {\"ContainerName\": \"<< parameters.container-name-for-code-deploy >>\", \"ContainerPort\": << parameters.container-port-for-code-deploy >>}}}}]}"}}' \
+                    --query deploymentId)
```

Comment:
What do you think of the following changes? That way, the `verify-revision-is-deployed` parameter should still be able to work if set to true.

```
DEPLOYED_REVISION="${CCI_ORB_AWS_ECS_REGISTERED_TASK_DFN}"
DEPLOYMENT_ID=$(aws deploy create-deployment ...)
```

and also adding `echo "Created CodeDeploy deployment: $DEPLOYMENT_ID"`